Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -1732,34 +1732,41 @@ static _WKAutoplayEvent toWKAutoplayEvent(WebCore::AutoplayEvent event)
 
 #if ENABLE(POINTER_LOCK)
 
-void UIDelegate::UIClient::requestPointerLock(WebPageProxy* page)
+void UIDelegate::UIClient::requestPointerLock(WebPageProxy* page, CompletionHandler<void(bool)>&& reply)
 {
     RefPtr uiDelegate = m_uiDelegate.get();
-    if (!uiDelegate)
+    if (!uiDelegate) {
+        reply(false);
```

Comment:
nit: I like "return reply(false);" because it's one line and doesn't require { }.  Some feel otherwise.